src/command_modules/azure-cli-monitor/azure/cli/command_modules/monitor/autoscale-parameters-template.json

```diff
@@ -0,0 +1,38 @@
+{
+  "autoscale_setting_resource_name": "autoscale setting resource name",
+  "location": "West US",
+  "tags": {},
+  "profiles": [
+    {
+      "name": "name of the profile",
+      "capacity": {
+        "minimum": "1",
+        "maximum": "5",
+        "default": "3"
+      },
+      "rules": [
+        {
+          "metric_trigger": {
+            "metric_name": "name of the metric",
+            "metric_resource_uri": "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/{providerName}/{resourceName}/{subResourceName}",
+            "time_grain": "(duration in ISO8601 format)PT5M",
+            "statistic": "Average or Min or Max or Sum",
+            "time_window": "(duration in ISO8601 format)PT45M",
+            "time_aggregation": "Average or Minimum or Maximum or Total or Count",
+            "operator": "Equals or NotEquals or GreaterThan or GreaterThanOrEqual or LessThan or LessThanOrEquals",
+            "threshold": 100
```

Comment:
I've verified that this is `float` type but properties of capacity has `string` type.